index.js

```diff
@@ -11,8 +11,8 @@ function format(obj) {
     : JSON.stringify(obj);
 }
 
-module.exports = function(hash) {
-  function visit(hash, prefix) {
+module.exports = function(hash, indent=0, newline_after_section=false) {
```

Comment:
Let's camelCase `newline_after_section` to `newlineAfterSection`.
```suggestion
module.exports = function(hash, indent=0, newlineAfterSection=false) {
```